api/doc.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package api defines the API cc-proxy exposes to clients, processes
+// connecting to the proxy AF_UNIX socket.
```

Comment:
I'd change this to the following to make it easier to read:}

"Package api defines the API cc-proxy exposes to clients (processes connecting to the AF_UNIX proxy socket)."